Tests/BootCamp.Chapter.Tests/Lesson3Tests.cs

```diff
@@ -56,7 +56,7 @@ public void PromptFloat_PrintsMessage_And_ReturnsFloat(string input, float expec
         {
             var consoleOutput = ConsoleStub.StubConsole(input);
 
-            var convertedInput = Checks.PromptFloat(PromptMessage);
+            var convertedInput = Lesson3.PromptFloat(PromptMessage);
```

Comment:
You probably changed this by auto-refactoring something. Be careful when doing so and let Visual Studio show you which files are going to be changed in which way!